test/scripts/scripts.test.js

```diff
@@ -81,12 +81,18 @@ describe('Loading', () => {
 
   it('Removes LCP block out of block list', async () => {
     const blocks = [...document.querySelectorAll('body > main > .section > [class]')];
-    await scripts.loadLCP(blocks);
+    await utils.loadLCP({ blocks });
     expect(blocks.length).to.equal(3);
   });
 
-  it('loadDelayed() test', () => {
-    scripts.loadDelayed();
+  it('loadDelayed() test', async () => {
+    const mod = await utils.loadDelayed(0);
+    expect(mod).to.exist;
+  });
+
+  it('modal test', () => {
+    const modal = document.querySelector('dialog');
```

Comment:
Is this really testing anything?

Reply:
It's testing that `loadModals` on `loadPage` fires and that there are no modals on page load. Admittedly, it's dumb, but it's a coverage thing.